pkg/exchange/coinbase/convert.go

```diff
@@ -41,24 +41,32 @@ func toGlobalOrderStatus(order *api.Order) types.OrderStatus {
 }
 
 func toGlobalOrder(cbOrder *api.Order) types.Order {
-	return types.Order{
+	order := types.Order{
 		SubmitOrder: types.SubmitOrder{
 			ClientOrderID: cbOrder.ClientOID,
+			Symbol:        toGlobalSymbol(cbOrder.ProductID),
 			Type:          toGlobalOrderType(cbOrder.Type),
 			Side:          toGlobalSide(cbOrder.Side),
 			Quantity:      cbOrder.Size,
 			Price:         cbOrder.Price,
 			StopPrice:     cbOrder.StopPrice,
 			TimeInForce:   toGlobalTimeInForce(cbOrder.TimeInForce),
 		},
-		Exchange:       types.ExchangeCoinBase,
-		Status:         toGlobalOrderStatus(cbOrder),
-		UUID:           cbOrder.ID,
-		OrderID:        FNV64a(cbOrder.ID),
-		OriginalStatus: string(cbOrder.Status),
-		CreationTime:   cbOrder.CreatedAt,
-		IsWorking:      isWorkingOrder(cbOrder.Status),
+		Exchange:         types.ExchangeCoinBase,
+		Status:           toGlobalOrderStatus(cbOrder),
+		UUID:             cbOrder.ID,
+		OrderID:          FNV64a(cbOrder.ID),
+		OriginalStatus:   string(cbOrder.Status),
+		CreationTime:     cbOrder.CreatedAt,
+		IsWorking:        isWorkingOrder(cbOrder.Status),
+		ExecutedQuantity: cbOrder.FilledSize,
 	}
+	if cbOrder.Status == api.OrderStatusDone {
+		order.UpdateTime = cbOrder.DoneAt
+	} else {
+		order.UpdateTime = types.Time(time.Now())
+	}
```

Comment:
Be careful that this could lead to an order is done before it has been partially filled.